File: python/talk.py
import requests

# Sample array
array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]

# Data that we will send in post request.
data = {'array': array}
print(array)

# The POST request to our node server
res = requests.post('http://127.0.0.1:3000/arraysum', json=data)

# Convert response data to json
returned_data = res.json()

print(returned_data)
result = returned_data['result']
print("Sum of Array from Node.js:", result)

File: node/talk.js

import fetch from 'node-fetch';
const text = { "text": "測試" };
const data = JSON.stringify(text);

const res_zhuyin = await fetch("http://127.0.0.1:5000/zhuyin", {
  method: 'POST',
  body: data,
	headers: {'Content-Type': 'application/json'}
});
console.log(res_zhuyin);

const dt = await res_zhuyin.json();
console.log(dt['result'].join(' '));